lib/components/map/itinerary-summary-overlay.tsx

```diff
@@ -57,7 +70,8 @@ const Card = styled.div`
     }
   }
   div {
-    margin-top: -0px!important;
+    margin-top: -0px !important;
```

Comment:
I doing think this `-` is doing anything!

Reply:
Good catch haah